spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    full_name { Faker::Name.name }
+    email { Faker::Internet.email }
+    password { Faker::Internet.password }
+    phone_number { Faker::PhoneNumber.cell_phone }
+    gender { Faker::Gender.types }
```

Comment:
Faker::Gender.types nó ra giá trị ntn ý e nhỉ, nhìn tnay giống nó in ra tất cả các giá trị available của types chứ nhỉ, in ra 1 mảng

Reply:
em đổi lại rồi ạ